src/orderbook.ts

```diff
@@ -230,7 +170,7 @@ function addOffer(offer: Offer, existingOffers: Map<number, Offer>) {
 }
 
 function sortOffersAscending(left: Offer, right: Offer) {
-  return left.price.toNumber() - right.price.toNumber();
+  return left.price.sub(right.price).toNumber();
```

Comment:
Maybe this should use Fraction comparisons? If you implement my proposed changes to `.gt` then also this should change.

Reply:
The sorting function needs to return a number:

> It is expected to return a negative value if first argument is less than second argument, zero if they're equal and a positive value otherwise.

I can also implement `lt` on Fraction and use a combination of `gt` and `lt`.

Reply:
I'd do it just for the peace of mind of not worrying about consistency between sorting and `.gt`. On the other end, for what I understand about float division you could have an inconsistent result only when your fractions have a numerator/denominator ratio of more than ~10^162, so maybe we don't want to worry about that.

Reply:
It's done! Thanks for the suggestions.